python/two-fer/two_fer.py

```diff
@@ -1,2 +1,6 @@
-def two_fer(name):
-    pass
+def two_fer(name = None):
+    if name == None:
```

Comment:
It is preferred to use `is None` to check whether variable is None rather than `==`